.github/workflows/ci.yml

```diff
@@ -65,9 +65,50 @@ jobs:
         if: ${{ env.SECRETS_AVAILABLE == 'true' }}
         run: sudo tailscale down
 
+  package:
+    name: Package compiles as a dependency
+    needs: build
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v1
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
```

Comment:
**CONCERN:** Shouldn't we use `14.x`? I thought that we should be using the latest LTS.

Reply:
`12.x` is used across the rest of the CI (was LTS at time of writing). Lets bump them all at the same time (in another PR).